projects/adrena/index.js

```diff
@@ -0,0 +1,61 @@
+const { sumTokens2, getConnection } = require("../helper/solana");
+const { PublicKey } = require("@solana/web3.js")
+const { decodeAccount } = require('../helper/utils/solana/layout')
+const BN = require("bn.js");
+
+async function staking() {
+  const connection = getConnection("solana");
+
+  const res = await connection.getMultipleAccountsInfo([
+    // ADX Staked Tokens
+    new PublicKey('9nD5AenzdbhRqWo7JufdNBbC4VjZ5QH7jzLuvPZy2rhb'),
+
+    // ALP Staked Tokens
+    new PublicKey('HYDnBK56NXcdaeKzVqTRBbm4aaRmfy2oErjrb1YfH7Zg'),
+  ]);
+
+  return {
+    // ADX
+    'solana:AuQaustGiaqxRvj2gtCdrd22PBzTn8kM3kEPEkZCtuDw': decodeAccount('tokenAccount', res[0]).amount.toString(),
+    
+    // ALP
+    'solana:4yCLi5yWGzpTWMQ1iWHG5CrGYAdBkhyEdsuSugjDUqwj': decodeAccount('tokenAccount', res[1]).amount.toString(),
```

Comment:
would skip this since the tokens backing it is already counted towards tvl

Reply:
Hello, we have a staking system where people can stake theirs ALP (pool token) in the program for some time 90day, 180d etc.

Same for our governance token (ADX).

I guess it depends what defilama is doing with the data, will follow your recommendation.

wdyt?

(thanks for review!)

Reply:
the reason is, when a user deposits 1 USDC into your protocol, it counts towards tvl, then they stake the ALP, now that same USDC is counted again under staking, do you see the issue?

Reply:
understandable! done ✅